src/components/atoms/Link/index.tsx

```diff
@@ -0,0 +1,30 @@
+import cx from 'classnames';
+import Link from 'next/link';
+import React from 'react';
+
+export type LinkProps = {
+  href: string;
+  title?: string;
+  style?: 'primary' | 'white' | 'gray';
+};
+
+const LinkButton: React.FC<LinkProps> = ({ href, title, style }) => {
+  return (
+    <div>
+      <Link href={href} passHref>
+        <a
+          className={cx('text-end text-xs hover:underline ', {
+            ' text-scrummyOrange-500 decoration-scrummyOrange-500 ':
```

Comment:
i invite you to think this using other props;

**example:** bgColor, text Color, size, etc...

if we use a unique prop called styled, this component has a very coupled style.